src/index.html

```diff
@@ -7,6 +7,390 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img src="images/logo.png" alt="Moyo Logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <ul class="main__list">
+        <li class="main__item">
+          <div class="card" data-qa="card" >
+            <img
+              src="images/imac.jpeg"
+              alt="APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)"
+              class="card__image"
+            >
+            <div class="card__content">
+              <h3 class="card__title">
+                APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)
+              </h3>
+              <p class="card__code">
+                Product code: 195434
+              </p>
```

Comment:
add blanks between multiline elements, but no need to add between one line elements